src/app/components/sites/pages/new/new.component.ts

```diff
@@ -72,13 +68,39 @@ class NewComponent extends FormTemplate<Site> {
   }
 }
 
-NewComponent.LinkComponentToPageInfo({
+@Component({
+  selector: "app-points-new",
```

Comment:
I thought we weren't using the `app` prefix for components

Reply:
`app` is the default prefix, ive been using `baw` for components which we may extract from the entire app later down the line. Although there is nothing preventing me from changing everything to use `baw` instead of `app`.

Reply:
app for only general components.

We're not extracting a point new component to something not specific to this app

Reply:
I don't follow